tests/mocks/asset.ts

```diff
@@ -0,0 +1,31 @@
+import BigNumber from 'bignumber.js';
+
+import { AssetToken } from '../../source/renderer/app/api/assets/types';
+
+export const zeroDecimalPlacesToken: AssetToken = {
+    policyId: '6b8d07d69639e9413dd637a1a815a7323c69c86abbafb66dbfdb1aa7',
+    assetName: '',
+    quantity: new BigNumber(6),
+    fingerprint: 'asset1cvmyrfrc7lpht2hcjwr9lulzyyjv27uxh3kcz0',
+    metadata: {
+        url: 'https://developers.cardano.org/',
+        name: 'ZeroDecimalCoin',
+        ticker: 'ZDP',
+        description: 'Zero decimal coin.'
+    },
+    recommendedDecimals: 0,
+    uniqueId: '6b8d07d69639e9413dd637a1a815a7323c69c86abbafb66dbfdb1aa7',
+    decimals: undefined,
+}
```

Comment:
Just a suggestion: Maybe it would be worth to extract the common properties to separate object and use it in both exported objects